iothub/device/src/DeviceClient.ConventionBasedOperations.cs

```diff
@@ -14,6 +15,11 @@ namespace Microsoft.Azure.Devices.Client
     /// <threadsafety static="true" instance="true" />
     public partial class DeviceClient : IDisposable
     {
+        /// <summary>
+        /// The <see cref="PayloadConvention"/> that the client uses for convention-based operations.
+        /// </summary>
+        public PayloadConvention PayloadConvention => InternalClient.PayloadConvention;
```

Comment:
This is exposed on the device client to help users build their writable property response.